src/app/services/user/user-identity-provider.service.ts

```diff
@@ -0,0 +1,26 @@
+import { Injectable } from "@angular/core";
+import { IIronIdentityProvider } from '../iron/i-iron-identity-provider';
+
+@Injectable({
+    providedIn: "root"
+})
+export class UserIdentityProviderService implements IIronIdentityProvider {
+    /**
+     * Request a signed JWT from the server for the current project and segment.
+     */
+    getJWT(userID): Promise<string> {
+        // TODO: Catch and gracefully handle invalid JWT
```

Comment:
This TODO should either be addressed in this PR or added as a card to the board.